cypress/integration/channel/message_permalink.js

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 6] */
+
+import users from '../../fixtures/users.json';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+let testChannel;
+let linkText;
+
+function createNewDMChannel(channelname) {
+    cy.get('#addDirectChannel').scrollIntoView().click();
+
+    cy.get('#selectItems').within(() => {
+        cy.get('input[type="text"]').scrollIntoView().type(channelname, {force: true});
+    });
+    cy.contains('.more-modal__description', channelname).click({force: true});
+    cy.get('#saveItems').click();
+}
+
+describe('Message permalink', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M13675-Copy a permalink and paste into another channel', () => {
+        const message = 'Hello' + Date.now();
+        const channelName = 'test-message-channel-1';
+
+        // # Create new DM channel with user's email
+        createNewDMChannel(users['user-2'].email);
```

Comment:
Does this step necessary?

Reply:
What alternate would you suggest for sending a message and copying permalink of message ?

Reply:
@composednitin I re-read the JIRA, and it looks like we just need to be on another channel, no need to be a DM or group channel.

Reply:
Hi @thekiiingbob ,
sorry for delayed response, was bit busy with some other projects.
i think we need to past permalink to new channel and we don't have any other channel, we need to create one

Reply:
Hi @thekiiingbob ,
sorry for delayed response, was bit busy with some other projects.
i think we need to past permalink to new channel and we don't have any other channel, we need to create one